### DISCUSSION TITLE
БЛЯДСКИЕ КОДЫ ПЕРЕСТАЛИ РАБОТАТЬ

### DISCUSSION THREAD

**Discussion by nixend-v8a** [ente-io/ente#3968] [Q&A] [answered]
ВСЕ КОДЫ ПЕРЕСТАЛИ МАТЬ ЕГО РАБОТАТЬ

Почта для связи - igtit666@gmail.com 

ой, заработали)))

**Comment by mrksbgg** [comment, marked as answer]
ща починим жди
